### ISSUE TITLE
redeploy webcam for traffic counting

### ISSUE THREAD

**Issue by jfilter** [codeforberlin/tickets#14]
Serious Machine Learning with Artificial Neural Networks: https://github.com/moovel/lab-opendatacam/

**Comment by derhuerst** [comment]
The board is ~400£. 😁

**Comment by jfilter** [comment]
Yes, not a beginner project. Maybe it‘s more for organizations such as Wikimedia.

**Comment by ulid000** [comment]
Michael Kleis (michael.kleis@gmail.com, https://www.meetup.com/Datacycle-Meetup-Fahrraddaten-fur-Berlin/members/263134183/?_locale=de-DE) presented his own development based on OpenCV and MobileNet at https://www.meetup.com/de-DE/Datacycle-Meetup-Fahrraddaten-fur-Berlin/ : https://docs.google.com/presentation/d/1IOxVlcVuxIJn2bHd793-Yg8_abxYj4biwLZCp1HAvkA/edit#slide=id.g4415e2c898_0_0
